hazelcast/include/hazelcast/client/internal/config/ConfigUtils.h

```diff
@@ -44,10 +44,6 @@ namespace hazelcast {
                         if (configPatternKey.get() != NULL) {
                             return configPatterns.get(*configPatternKey);
                         }
-                        if ("default" != itemName && itemName.find("hz:") != 0) {
```

Comment:
Why did we remove this part?

Reply:
Since it was a finest level message and otherwise we need to provide logger to this static method, i thought it would be easier t just not print the message. Also, it is being called from ClientConfig which has no associated logger with it, since the logger is per HazelcastClient instance but ClientConfig is created before the instance is created. The method returns null pointer which actually may imply the same thing except that this log is only for when item name starts with "hz:". Do you think that this log should exist and worth the effort?

Reply:
On java we use a different logger here via static `Logger.getLogger()`. Can't we do something similar ? 
I mean the logs does not have to contain instance name. It seems we can't also apply users log option also. Not sure if we have to include  this part, I was just comparing to java. Decision is up to you.

Reply:
Exactly, we do not have a static logger. Yes, a static logger can be added but what will be its configuration? Write to file? Write to stdout? What will be the log level for this logger and it won't be related to any instance? I believe for Java it uses for example default logger level for log42j. I think it is better that we do not use static loggers like this.